test/e2e/catalog_e2e_test.go

```diff
@@ -1451,6 +1451,7 @@ var _ = Describe("Starting CatalogSource e2e tests", func() {
 				}).Should(Succeed())
 			})
 			It("The registry pod fails to become come up because of lack of permission", func() {
+				Skip("This test will not work until catalogSources run in restricted mode by default")
```

Comment:
Non-blocking for follow-up. It seems like we could still exercise this case by explicitly setting the securityContextConfig on the catalog source rather than depending on defaulting?

Reply:
That makes sense to me, the test will now focus on ensuring that restricted catalogSources can't run with non-root images.